Compiler/runtime/zeromqimpl.c

```diff
@@ -71,22 +58,12 @@ void* ZeroMQ_initialize()
   // create the file path
   const char* tempPath = SettingsImpl__getTempDirectoryPath();
 #if defined(__MINGW32__) || defined(_MSC_VER)
-  if (zeroMQFileSuffix != NULL) {
-    zeroMQFilePath = (char*)malloc(strlen(tempPath) + strlen("/openmodelica.port.") + strlen(zeroMQFileSuffix) + 1);
-    sprintf(zeroMQFilePath, "%s/openmodelica.port.%s", tempPath, zeroMQFileSuffix);
-  } else {
-    zeroMQFilePath = (char*)malloc(strlen(tempPath) + strlen("/openmodelica.port") + 1);
-    sprintf(zeroMQFilePath, "%s/openmodelica.port", tempPath);
-  }
+  zeroMQFilePath = (char*)malloc(strlen(tempPath) + strlen("/openmodelica.port") + strlen(zeroMQFileSuffix) + 1);
+  sprintf(zeroMQFilePath, "%s/openmodelica.port%s", tempPath, zeroMQFileSuffix);
```

Comment:
This broke OMPython. You removed a `.` after port.

Reply:
It doesn't break it. The suffix was fixed. If it was ".port", it still will be. If it was "port", it will not contain the dot.

Reply:
No it does. Previously it was that if there is no suffix then just use `openmodelica.port` and if suffix then `openmodelica.port.suffix`. You changed it to `openmodelica.portsuffix`. Fix this or update OMPython.